we don't have such include file and it seems it is not needed 
see also: https://github.com/mate-desktop/mate-screensaver/issues/272

diff --git a/helper/pam-helper.c b/helper/pam-helper.c
index 9c30c46..8291d63 100644
--- a/helper/pam-helper.c
+++ b/helper/pam-helper.c
@@ -33,7 +33,6 @@
  */
 
 #include <security/pam_appl.h>
-#include <security/_pam_macros.h>
 
 #include <sys/types.h>
 #include <stdarg.h>
